library/src/main/scala/scala/scalajs/js/UndefOr.scala

```diff
@@ -158,6 +158,19 @@ final class UndefOrOps[A](val self: UndefOr[A]) extends AnyVal {
   @inline final def withFilter(p: A => Boolean): WithFilter[A] =
     new WithFilter(self, p)
 
+  /** Tests whether the $option contains a given value as an element.
+   *
+   *  `x.contains(y)` differs from `x == y` only in the fact that it will
+   *  return `false` when `x` and `y` are both `undefined` (which is also
+   *  `()`).
```

Comment:
I'm not sure the comment in parens actually adds something here. If the comment is needed, it should probably be on the class level. Here it is confusing, as it raises the question: "Why is () special in `contains`?"